server/swap/go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/decred/dcrdex/server/swap
+
+go 1.13
+
+require (
+	github.com/decred/dcrdex/server/account v0.0.0-20190913133319-627e24311406
+	github.com/decred/dcrdex/server/asset v0.0.0-00010101000000-000000000000
+	github.com/decred/dcrdex/server/comms v0.0.0-00010101000000-000000000000
+	github.com/decred/dcrdex/server/comms/rpc v0.0.0-00010101000000-000000000000
+	github.com/decred/dcrdex/server/matcher v0.0.0-00010101000000-000000000000
```

Comment:
If possible try to `go get` (with replace temporarily removed) valid versions in here, even though we are replacing them.  I know some of them may be broken because they require modules with invalid versions too, but that's why we gotta get these out if possible.